templates/ansible/resource.erb

```diff
@@ -266,15 +266,30 @@ def update_fields(module, request, response):
 
 <% update_props.each do |key, props| -%>
```

Comment:
Does standard patch / put code live somewhere else for Ansible? TF has a concept of resource update vs custom (field) update, and I think this corresponds to field update. This part is just curiosity.

Reply:
Right now, all of the update code is mixed into this very large if/else block. I've been wanting to clean this up, but it's not very high on my priority list.